src/AI.hpp

```diff
@@ -51,9 +52,14 @@
         unsigned int maxReconnect;
 
         libclient::LibClient libClientHandler;
-        spy::MatchConfig matchConfig;
+        std::optional<spy::MatchConfig> matchConfig;
```

Comment:
Willst du die Objekte hier in der KI haben oder soll man das in die LibClient auslagern?
Könnte ich evtl. dann auch mit dem Client nutzen

Reply:
Die werden ja aus der LibClient geholt und assigned (im onMetaInformation callback). Habe das hauptsächlich in der KI, damit ich weiß, ob ich die ScenarioConfig habe oder nicht.
Oder an welche Verwendung hast du im Client gedacht?